Simplified/NYPLAnnotations.swift

```diff
@@ -12,177 +12,158 @@ final class NYPLAnnotations: NSObject {
+    {
+      if book.annotationsURL != nil {
+        
+        var request = URLRequest(url: book.annotationsURL)
+        request.httpMethod = "GET"
+        
+        for (headerKey, headerValue) in NYPLAnnotations.headers {
+          request.setValue(headerValue, forHTTPHeaderField: headerKey)
+        }
+        
+        let dataTask = URLSession.shared.dataTask(with: request) { (data, response, error) in
+          
+          if error != nil {
+            completionHandler(nil, error as? NSError)
+          } else {
+            
+            let jsonData: [String:Any]?
+            do {
+              jsonData = try JSONSerialization.jsonObject(with: data!, options: []) as? [String:Any]
+            } catch {
+              Log.error(#file, "JSON could not be created from data.")
+              completionHandler(nil, nil)
+              return
+            }
+            
+            if let json = jsonData {
+              let total:Int = json["total"] as! Int
+              if total > 0
+              {
+                let first = json["first"] as! [String:AnyObject]
+                let items = first["items"] as! [AnyObject]
+                for item in items
+                {
+                  let target = item["target"] as! [String:AnyObject]
+                  let source = target["source"] as! String
+                  if source == book.identifier
+                  {
+                    let selector = target["selector"] as! [String:AnyObject]
+                    let value = selector["value"] as! String
+                    
+                    completionHandler(value as String!, error as? NSError)
+                    Log.info(#file, value)
+                  }
+                }
+              } else {
+                completionHandler(nil, error as? NSError)
+              }
+            } else {
+              completionHandler(nil, error as? NSError)
+            }
+          }
+        }
+        dataTask.resume()
+      }
+    }
   }
   
-  private class var headers: [String:String]
+  class var headers: [String:String]
   {
     let authenticationString = "\(NYPLAccount.shared().barcode!):\(NYPLAccount.shared().pin!)"
     let authenticationData:Data = authenticationString.data(using: String.Encoding.ascii)!
-    let authenticationValue = "Basic \(authenticationData.base64EncodedString(options: Data.Base64EncodingOptions.lineLength64Characters)))"
+    let authenticationValue = "Basic \(authenticationData.base64EncodedString(options: Data.Base64EncodingOptions.lineLength64Characters))"
     
-    return ["Authentication" : "\(authenticationValue)"]
+    return ["Authorization" : "\(authenticationValue)",
+            "Content-Type" : "application/json"]
+  }
+  
+  class func iso8601CurrentTime() -> String
+  {
+    let dateFormatter = DateFormatter()
+    let enUSPosixLocale = Locale(identifier: "en_US_POSIX")
+    dateFormatter.locale = enUSPosixLocale
+    dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZZ"
```

Comment:
if the result is the same, i am ok with using  RFC3339String, we just need to make sure it is the same on both iOS and Android